src/plugins_types.h

```diff
@@ -181,6 +181,9 @@ void lysc_prefixes_free(struct lysc_prefix *prefixes);
  * must function as expected.
  *
  * Note that the \p value string is not necessarily zero-terminated. The provided \p value_len is always correct.
+ * Note that dynamically allocated value can be added to dictonary by function zero copy. This means that all store function
```

Comment:
Not sure if this note is really needed, basically the same text is in the description of `LY_TYPE_STORE_DYNAMIC`.

Reply:
I wouldn't connect the note with the dictionary and its zero copy functionality, but the general note regarding the expected behavior of the callback seems useful to me.

Reply:
Yes, I was referring to the first sentence. The other one is fine.